### DISCUSSION TITLE
Welcome to cleanlab Discussions!

### DISCUSSION THREAD

**Discussion by cgnorthcutt** [cleanlab/cleanlab#56] [General]
## 👋 Welcome!
  We’re using Discussions to help you learn more about cleanlab and connect with other members of our community. We hope that you:
  * Ask questions about cleanlab.
  * Share simple code snippets to solve various problems (e.g. PU learning, multi-label noisy learning, etc).
  * Engage with other community members.
  * Be kind and open-minded with one another -- this is a community we
  build together 💪.

**Comment by artem-math** [comment]
Dear Developers! Thank you for this great software and research =) I guess that there is a typo in your tutorial on PU learning here https://github.com/cgnorthcutt/cleanlab#pu-learning-with-cleanlab
You have
lnl = LearningWithNoisyLabels(clf=LogisticRegression(pulearning=pu_class))
but it should be
lnl = LearningWithNoisyLabels(clf=LogisticRegression(), pulearning=pu_class)
right?
Thank you!

**Reply by cgnorthcutt** [reply]
Updated here: https://github.com/cgnorthcutt/cleanlab/commit/4a2bdc5e1b2f7220779a1423f6f244e3e2d77a54

Thank you!

**Comment by NorthyFN** [comment]
Im just starting to get the hang of this and read on how it works. But right now from the first results it looks like pure black magic... So thank you for this!! ;)

**Reply by cgnorthcutt** [reply]
Glad it's working for you! Lots of updates to come over the next year :)
